internal/tiltfile/files.go

```diff
@@ -372,3 +373,84 @@ func (s *tiltfileState) listdir(thread *starlark.Thread, fn *starlark.Builtin, a
 
 	return starlark.NewList(ret), nil
 }
+
+func (s *tiltfileState) jsonDecode(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var jsonString starlark.String
+	if err := starlark.UnpackArgs(fn.Name(), args, kwargs, "json", &jsonString); err != nil {
+		return nil, err
+	}
+
+	var decodedJSON interface{}
+
+	if err := json.Unmarshal([]byte(jsonString), &decodedJSON); err != nil {
+		return nil, err
+	}
+
+	return convertJsonToStarlark(decodedJSON)
+}
+
+func (s *tiltfileState) readJson(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var path starlark.String
+	if err := starlark.UnpackArgs(fn.Name(), args, kwargs, "path", &path); err != nil {
+		return nil, err
+	}
+
+	localPath, err := s.localPathFromSkylarkValue(path)
+	if err != nil {
+		return nil, fmt.Errorf("Argument 0 (path): %v", err)
+	}
+
+	contents, err := s.readFile(localPath)
+	if err != nil {
+		return nil, err
+	}
+
+	var decodedJSON interface{}
+
+	if err := json.Unmarshal(contents, &decodedJSON); err != nil {
+		return nil, err
```

Comment:
I think it's worth writing a test for invalid JSON and then wrapping this error to make sure it's useful.

When I try to read an invalid JSON file, it currently looks like the Tiltfile itself has a syntax error:
```
       2019/03/13 11:21:40 Beginning Tiltfile execution
       Traceback (most recent call last):
         /Users/matt/go/src/github.com/windmilleng/servantes/Tiltfile:2: in
       <toplevel>
       Error: invalid character '\n' in string literal
```